nav2_map_server/include/nav2_map_server_3D/map_io_3D.hpp

```diff
@@ -0,0 +1,86 @@
+//
+// Created by shivam on 7/5/20.
+//
+
+#ifndef NAV2_MAP_SERVER_SRC_MAP_IO_3D_HPP_
+#define NAV2_MAP_SERVER_SRC_MAP_IO_3D_HPP_
+
+#include <string>
+#include <vector>
+
+#include "sensor_msgs/msg/point_cloud2.hpp"
+#include "geometry_msgs/msg/transform.hpp"
+
+namespace nav2_map_server {
+
+/**
+ * @brief nav2_map_server_3D namespace containing the utilities to use point cloud mapss
+ */
+namespace nav2_map_server_3D {
+
+/**
+ * @brief parameters that will be populated while reading a YAML file
+ */
+struct LoadParameters_3D {
+  std::string pcd_file_name;
+  std::vector<float> view_point;
+};
+
+typedef enum {
```

Comment:
enum class instead

Reply:
I used this because of its use in map_io.cpp (code equity) 
should I change it??